test/exception_notifier/google_chat_notifier_test.rb

```diff
@@ -160,12 +160,12 @@ def header
     [
       '',
       'Application: *dummy*',
-      'An *ArgumentError* occured.',
+      'A *ArgumentError* occurred.',
```

Comment:
I'd adjust the code to return `An` in this case.

Reply:
changed here: df864ea68405a49a213fa0d0ca839983af31d0cc